client/views/view_research.cpp

```diff
@@ -20,12 +20,16 @@
 #include <QPainter>
 #include <QPushButton>
 #include <QScrollArea>
+#include <QStringLiteral>
 #include <QTimer>
+#include <QToolButton>
```

Comment:
Why `QToolButton` when we already have `QPushButton` for the refresh button?

Reply:
Because Louis said:

> I would try a design with an icon-only `QToolButton`...

As QT is still uncharted territory for me, I can't say whether there is any objective reason to prefer one button class over the other.

Reply:
QPushButton is rectangular with text, QToolButton is square and icon-only.